cashu/wallet/api/router.py

```diff
@@ -32,7 +32,7 @@ async def load_mint(wallet: Wallet, mint: Optional[str] = None):
     try:
         await wallet.load_mint()
     except Exception as e:
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
+        raise Exception(str(e))
```

Comment:
This block can be replaced by simply `wallet.load_mint()` without `try: ... except: ...` – achieves the same thing.